host/greatfet/commands/greatfet_logic.py

```diff
@@ -38,20 +40,43 @@ def main():
             print("No GreatFET board found!", file=sys.stderr)
         sys.exit(errno.ENODEV)
 
-    device.apis.logic_analyzer.start()
+    # pulseview metadata
```

Comment:
It'd be nice to have this also support outputting raw binary, as it did, so it can more easily be consumed by other tools. Can that be set by the command-line arguments?

Reply:
So optionally saving a raw binary data file separate from the pulseview archive?